resources/markdown/who_audit_l2.md

```diff
@@ -0,0 +1,282 @@
+- Display Question #1 of the (WHO) AUDIT-C
+- Record and display the response to Question #1
+</pre>
+
+----------------------------------------------------------------------
+## LOGIC PATH #2 (stop screening):
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'NO'
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- End screening
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 3: Consider administering Question #2  and Question #3  of the (WHO) AUDIT-C: 
+
+## LOGIC PATH #1: Continue screening for women, men > 65 y.o., or individuals whose sex at birth is unknown:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND NOT most recent (WHO) AUDIT-C Question #1 response is 'Never'
+AND
+     Female
+     OR Male > 65 years old
+     OR Sex at birth is unknown
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Continue to display (WHO) AUDIT-C screening instructions
+- Continue to display U.S. standard drink size graphic
+- Display Question #2 and Question #3 of the (WHO) AUDIT-C (Question #3 = How often do you have four or more drinks on one occasion?)
+- Record and display the response to Question #2  and Question #3
+- Calculate (WHO) AUDIT-C score
+- Record and display (WHO) AUDIT-C score
+</pre>
+
+----------------------------------------------------------------------
+## LOGIC PATH #2: Continue screening for men <= 65:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND NOT most recent (WHO) AUDIT-C Question #1 response is 'Never'
+AND Male <= 65 years old
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Continue to display (WHO) AUDIT-C screening instructions
+- Continue to display standard drink size graphic
+- Display Question #2  and Question #3 of the (WHO) AUDIT-C (Question #3 = How often do you have five or more drinks on one occasion?)
+- Record and display responses to Question #2 and Question #3
+- Calculate (WHO) AUDIT-C score
+- Record and display (WHO) AUDIT-C score
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 4: Consider presenting Question #9 and Question #10 of the (WHO) AUDIT when not administering the full (WHO) AUDIT: 
+
+## LOGIC PATH #1: Jump from Question #1 to Question #9 & Question #10 OR jump from end of (WHO) AUDIT-C to Question #9 & Question #10:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND
+     Most recent (WHO) AUDIT-C Question #1 response is 'Never'
```

Comment:
Ginny: For now, I left the following phrase in, but perhaps we could remove it since we are reasoning over the Q1 response in other phrases. (AND NOT most recent (WHO) AUDIT-C Question #1 response "Never"). Do you have strong feelings about this?

Reply:
GM - Haha I was looking at that and wondering if it was needed, and THEN saw your question. I don't suppose it hurts to have it there, but it seems cleaner not to? 
SS: I removed the NOT NEVER clause as we originally thought. It makes the logic a bit more clear that way. Thanks!